aten/src/ATen/core/dispatch/KernelRegistration.h

```diff
@@ -25,22 +24,19 @@ namespace c10 {
  *
```

Comment:
Re line above; you shouldn't refer to KernelRegistrationBuilder directly either, right?